### ISSUE TITLE
[Help] More Localization Support

### ISSUE THREAD

**Issue by TartaricAcid** [ZeroNoRyouki/BigReactors#117]
I found that a lot of gui's descriptions seem use hard code, can you add more localization for it?

**Comment by ZeroNoRyouki** [comment]
Yeah, there is a lot of stuff still hardcoded. I'll look into this after the 1.12 version is stable
